ddtrace/_trace/_span_pointer.py

```diff
@@ -53,3 +54,11 @@ def __init__(
     def __post_init__(self):
         # Do not want to do the trace_id and span_id checks that SpanLink does.
         pass
+
+
+def _standard_hashing_function(*elements: bytes) -> str:
+    if not elements:
+        raise ValueError("elements must not be empty")
```

Comment:
We have to make sure that we do everything possible to avoid crashing user applications. With that in mind, can we switch to an error log and either return an empty string or some other value that would keep us from crashing down the line.

Reply:
can we keep the exception and agree to always catch it further up the stack?

Reply:
alright, per our discussion on slack, i'm going to make sure we don't throw exceptions.